airbyte-db/lib/src/main/resources/jobs_database/Attempts.yaml

```diff
@@ -25,8 +25,6 @@ properties:
     type: ["null", object]
   status:
     type: string
-  temporal_workflow_id:
-    type: ["null", string]
```

Comment:
This schema is used for validation in the file-based migration system, which does not know the existence of this property.